.circleci/config.yml

```diff
@@ -121,50 +134,95 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+      - build_i18n:
           filters:
             tags:
               only: /^v.*/
+            branches:
+              only:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
       - unit_test:
           filters:
-            tags:
-              only: /^v.*/
+            branches:
+              ignore:
```

Comment:
The require of `build` should propagate the filters (that is, you don't need to re-include them here as it will not go through the required `build` job unless it meets the filter). Is there another reason we need to keep this?

Reply:
The [docs](https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag) made it seem like we need to include the filter at all levels. I'll do a test without them and see if it works. Are you suggesting that they propagate from build to unit_test?

Reply:
Ah I see, so this is the case of `tags` default being "don't run any tags", so it must be included on every level. However, the default for `branches` is all branches, and since only branches that meet the prior required step (`build`) pass through, using all branches works (Like how previously in our config, we did not copy the filters from the previous steps)

Reply:
Ah, I missed that distinction between how the `tags` default is to not run and how the `branches` default is to run for everything. So if I'm understanding correctly, I can remove the branch filters from the build and unit test jobs and only have the branch filters for the deploy jobs.

Reply:
Update: I do need the branch filters for the i18n build step but not the uint test step